go/client/chat_cli_ui.go

```diff
@@ -307,6 +307,16 @@ func (c *ChatCLIUI) ChatSearchConvHits(ctx context.Context, arg chat1.ChatSearch
 	return nil
 }
 
+func (c *ChatCLIUI) ChatSearchTeamHits(ctx context.Context, arg chat1.ChatSearchTeamHitsArg) error {
+	if c.noOutput {
+		return nil
+	}
+	for _, hit := range arg.Hits.Hits {
+		_ = c.terminal.Output(fmt.Sprintf("Team: %s found with matching name\n", hit.Name))
```

Comment:
Maybe we should give an indication that these are suggestions for the user to join and offer a command for them to move forward.

Reply:
Sure, happy to.  Any suggestions on a short explanation string that would fit in 80 chars (ideally)?  Maybe it needs to be over two lines, something like:

```sh
Team: keybasefriends found with matching name
  You can join this open team with `keybase team request-access keybasefriends`
```

Reply:
Ok I went with my suggestion above, hopefully that's better!

Reply:
Sounds good!